stable/datacube-ows/values.yaml

```diff
@@ -35,14 +35,7 @@ ows:
   autoscaling: True
   annotations:
     iam.amazonaws.com/role: kubernetes-wms
-  livenessProbe:
-    exec:
-      command:
-      - "/bin/sh"
-      - "-c"
-      - "curl --silent 'http://localhost:8000/?service=WMS&request=GetCapabilities&version=1.3.0' | grep -q -F '<Layer queryable=\"1\">'"
-    initialDelaySeconds: 30
-    periodSeconds: 30
+  livenessProbe: {}
```

Comment:
Perhaps this should be a readiness probe. Or slow down the frequency of this probe. This is also a relatively expensive probe and overwritten in the helm values ?

Reply:
added `startupProbe` I am still not convinced this curl does anything, i mean it executes it but it might never fail.